src/main/java/org/smof/annnotations/SmofPartialIndex.java

```diff
@@ -0,0 +1,17 @@
+package org.smof.annnotations;
```

Comment:
Is this annotation still needed?

Reply:
No we dont need that I will delete that and improve the pr.